swapi/src/app/sign-up/utils/sign-up.utils.ts

```diff
@@ -0,0 +1,8 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from "@angular/forms";
+
+export const passwordMatchValidator: ValidatorFn = (control: AbstractControl): ValidationErrors | null => {
+  const password = control.get('password');
+  const confirmPassword = control.get('confirmPassword');
+
+  return password && confirmPassword && password.value === confirmPassword.value ? { identityRevealed: true } : null;
```

Comment:
not sure i saw where do you use identityRevealed

Reply:
also not sure i saw, you display that passwords are not matched error to user

Reply:
I'm going to use the code in `password` and `confirmPassword` fields cross validation. just haven't added the error message

Reply:
This validator has reverse logic it works exactly opposite way compared to what we want here

Reply:
updated